### PR TITLE
Update all ra tooltips

### PR DESCRIPTION
**Pull request by abcdefg30**
"Follow-up" of #7967.

### CODE REVIEW

File: mods/ra/rules/infantry.yaml [hunk 1]
```diff
@@ -220,7 +220,7 @@ SPY:
 	DisguiseToolTip:
 		Name: Spy
 		GenericName: Soldier
-		Description: Infiltrates enemy structures for intel or\nsabotage. Exact effect depends on the\nbuilding infiltrated.\n  Strong vs Nothing\n  Weak vs Everything\n  Special Ability: Disguised
+		Description: Infiltrates enemy structures for intel or\nsabotage. Exact effect depends on the\nbuilding infiltrated.\nLooses disguise when attacking.\n  Strong vs Infantry\n  Weak vs Vehicles, Aircraft\n  Special Ability: Disguised
```
Comment (ScottNZ) [inline mods/ra/rules/infantry.yaml]: Loses

File: mods/ra/rules/aircraft.yaml [hunk 2]
```diff
@@ -230,7 +230,7 @@ HELI:
 		Cost: 2000
 	Tooltip:
 		Name: Longbow
-		Description: Helicopter Gunship with AG Missiles.\n  Strong vs Tanks\n  Weak vs Infantry
+		Description: Helicopter Gunship with AG and AA Missiles.\n  Strong vs Tanks, Aircraft\n  Weak vs Infantry
```
Comment (penev92) [inline mods/ra/rules/aircraft.yaml]: how about "Helicopter gunship armed with multi-purpose missiles."

File: mods/ra/rules/aircraft.yaml [hunk 3]
```diff
@@ -277,7 +277,7 @@ HIND:
 		Cost: 1350
 	Tooltip:
 		Name: Hind
-		Description: Helicopter Gunship with Chainguns.\n  Strong vs Infantry, Light Vehicles.\n  Weak vs Tanks
+		Description: Helicopter Gunship with Chainguns.\n  Strong vs Infantry, Light armor.\n  Weak vs Tanks, Aircraft
```
Comment (penev92) [inline mods/ra/rules/aircraft.yaml]: Drop a few upper-case letters and maybe use "armed with dual chainguns".

File: mods/ra/rules/infantry.yaml [hunk 4]
```diff
@@ -75,7 +75,7 @@ E2:
 		Cost: 160
 	Tooltip:
 		Name: Grenadier
-		Description: Infantry armed with grenades. \n  Strong vs Buildings, Infantry\n  Weak vs Vehicles
+		Description: Infantry armed with grenades. \n  Strong vs Buildings, Infantry\n  Weak vs Vehicles, Aircraft
```
Comment (penev92) [inline mods/ra/rules/infantry.yaml]: Redundant ' ' before '\n'.

**Comment by obrakmann** [comment]
It would be nice if all those instances where a unit is said to be "weak" against something when it can't even shoot at it at all could be cleaned up. Like "weak against aircraft" for the yak, for example. That implies to me that it does at least a little damage, not that it doesn't do anything at all.

**Comment by Phrohdoh** [comment]
I've made that argument before but it was outvoted.

File: mods/ra/rules/structures.yaml [hunk 5]
```diff
@@ -977,7 +977,7 @@ SILO:
 		Cost: 150
 	Tooltip:
 		Name: Silo
-		Description: Stores excess refined Ore and gems.
+		Description: Stores excess refined\nOre and gems.
```
Comment (Mailaender) [inline mods/ra/rules/structures.yaml]: Gems is in title case in the line above.

**Comment by Mailaender** [comment]
:+1: This corrects the tooltip it in several cases.

File: mods/ra/rules/structures.yaml (hunk 5 above)
Comment (abcdefg30) [inline mods/ra/rules/structures.yaml]: Fixed.

**Comment by penev92** [comment]
Looks good to me :+1:

**Comment by penev92** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/813394dcbe4be2a11bf07e5d6ce073a7bae2ff54...cb45ff5db15d738468c3e3c2320d35b1cbba5121)

**Pull request merged**